example7/spdx/example7-golang.spdx.json

```
@@ -15,30 +15,30 @@
   ],
```

I can't type the suggestion here, but its missing one package in the document describes field:
```suggestion
,
   "SPDXRef-go-compiler"
  ],
```